azure/table_azure_mysql_server.go

```diff
@@ -0,0 +1,294 @@
+			},
+			{
+				Name:        "master_server_id",
+				Description: "The master server id of a replica server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerProperties.MasterServerID"),
+			},
+			{
+				Name:        "minimal_tls_version",
+				Description: "Enforce a minimal Tls version for the server. Possible values include: 'TLS10', 'TLS11', 'TLS12', 'TLSEnforcementDisabled'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerProperties.MinimalTLSVersion").Transform(transform.ToString),
+			},
+			{
+				Name:        "public_network_access",
+				Description: "Indicates whether or not public network access is allowed for this server. Value is optional but if passed in, must be 'Enabled' or 'Disabled'. Possible values include: 'Enabled', 'Disabled'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerProperties.PublicNetworkAccess").Transform(transform.ToString),
+			},
+			{
+				Name:        "replica_capacity",
+				Description: "The maximum number of replicas that a master server can have.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("ServerProperties.ReplicaCapacity"),
+			},
+			{
+				Name:        "replication_role",
+				Description: "The replication role of the server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerProperties.ReplicationRole"),
+			},
+			{
+				Name:        "sku_capacity",
+				Description: "The scale up/out capacity, representing server's compute units.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("Sku.Capacity"),
+			},
+			{
+				Name:        "sku_family",
+				Description: "The family of hardware.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Sku.Family"),
+			},
+			{
+				Name:        "sku_name",
+				Description: "The name of the sku. For example: 'B_Gen4_1', 'GP_Gen5_8'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Sku.Name"),
+			},
+			{
+				Name:        "sku_size",
+				Description: "The size code, to be interpreted by resource as appropriate.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Sku.Size"),
+			},
+			{
+				Name:        "sku_tier",
+				Description: "The tier of the particular SKU. Possible values include: 'Basic', 'GeneralPurpose', 'MemoryOptimized'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Sku.Tier").Transform(transform.ToString),
+			},
+			{
+				Name:        "ssl_enforcement",
+				Description: "Enable ssl enforcement or not when connect to server. Possible values include: 'Enabled', 'Disabled'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServerProperties.SslEnforcement").Transform(transform.ToString),
+			},
+			{
+				Name:        "storage_auto_grow",
```

Comment:
```suggestion
				Name:        "storage_autogrow",
```

Reply:
As per terraform it auto grow is a separate word
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/mysql_server#auto_grow_enabled